core/src/main/java/com/contentful/vault/SyncConfig.java

```diff
@@ -90,6 +92,17 @@ public Builder setSpaceId(String spaceId) {
       return this;
     }
 
+    public Builder setEnvironment(String environment) {
+      if (client != null) {
+        throw new IllegalStateException(
+            "Do not set an environment, when a client is already set. User either environment " +
```

Comment:
@miron4dev I'm not sure I understand what this message means, is `User` the correct word at the start of the second sentence? Can you please explain what the message tries to say here?

Reply:
You're right. It's a typo.